Aspire Budgeting/Facilities/StateManager.swift

```diff
@@ -26,13 +26,15 @@ final class StateManager: ObservableObject {
   private var localAuthObserver: NSObjectProtocol?
   private var backgroundObserver: NSObjectProtocol?
   private var defaultSheetObserver: NSObjectProtocol?
+  private var logoutObserver: NSObjectProtocol?
   
   private lazy var transitions: [State: Set<State>] = {
     var transitions = [State: Set<State>]()
     
     transitions[.loggedOut] = [.verifiedGoogleUser]
     transitions[.verifiedGoogleUser] = [.authenticatedLocally, .localAuthFailed]
-    transitions[.authenticatedLocally] = [.localAuthFailed, .hasDefaultSheet, .needsLocalAuthentication]
+    transitions[.authenticatedLocally] = [.localAuthFailed, .hasDefaultSheet, .needsLocalAuthentication, .loggedOut]
+    //TODO: hasDefaultSheet transition is not used at all. Should be removed.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/91d35b9b1048725a3fbe3d56ea12293e8e0cbfb4/Aspire Budgeting/Facilities/StateManager.swift#L37"></span>TODOs should be resolved (hasDefaultSheet transition is …).<br />
<code>todo</code> <code>StateManager.swift:37</code></td>
    </tr>
  </tbody>
</table>